sui-execution/latest/sui-adapter/src/programmable_transactions/linkage_view.rs

```diff
@@ -40,6 +48,8 @@ pub struct LinkageView<'state> {
     /// Cache of past package addresses that have been the link context -- if a package is in this
     /// set, then we will not try to load its type origin table when setting it as a context (again).
     past_contexts: RefCell<HashSet<ObjectID>>,
+    /// Modules published for the session using this LinkageView.
+    published_modules: BTreeMap<ModuleId, Vec<u8>>,
```

Comment:
Chatted offline about consolidating this in `ExecutionContext`.

Reply:
Created a `SuiDataStore` that works with `ExecutionContext` to use new packages